webcompat/static/css/development/components/issue.css

```diff
@@ -164,15 +161,31 @@
   margin:4em 0 0;
 }
 
-/* labeleditor */
+/* labeleditor wrapper */
 .wc-Issue-labelEditor {
   position: relative;
-  display: inline-block;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 0 .625em 0;
 }
 
 @media all and (max-width: 23.43em) {
-
   .wc-Issue-labelEditor {
     position: static; /* position of label_editor not depend of this class */
   }
 }
+
+@media all and (min-width: 48.13em) {
+  .wc-Issue-aside {
+    float: right;
+    width: 30%;
+  }
+}
+
+@media all and (min-height: 40em), all and (min-width: 48.13em) {
+  .wc-Issue-aside {
+    position: sticky;
```

Comment:
@miketaylr don't know if a polyfill is necessary. Chrome, Safari and Firefox support sticky, but not edge, what do you think?